tueplots/figsizes.py

```diff
@@ -182,6 +182,36 @@ def jmlr2001(
     )
 
 
+def tmlr2023(
+    *,
+    rel_width=1.0,
+    nrows=1,
+    ncols=2,
+    constrained_layout=True,
+    tight_layout=False,
+    height_to_width_ratio=_GOLDEN_RATIO,
+    pad_inches=_PAD_INCHES,
+):
+    """TMLR figure size.
+
+    Source: https://www.overleaf.com/latex/templates/tmlr-journal-submissions/mwchznnhdtwx
```

Comment:
Should we link

https://www.jmlr.org/tmlr/author-guide.html

instead of overleaf? .

Reply:
yeah, you're right.